blog/en/Implement-Milvus-CLI-by-Python-Click.md

```diff
@@ -0,0 +1,830 @@
+
+#### Build
+
+1. Upgrade the `build` package: `python3 -m pip install --upgrade build`
+
+2. Run build: `python -m build --sdist --wheel --outdir dist/ .`
+3. Two files will be generated under the `dist/` directory:
+
+```shell
+dist/
+  example_package_YOUR_USERNAME_HERE-0.0.1-py3-none-any.whl
+  example_package_YOUR_USERNAME_HERE-0.0.1.tar.gz
+```
+
+### Publish release
+
+Refer to: https://packaging.python.org/tutorials/packaging-projects/#uploading-the-distribution-archives
+
+1. Upgrade `twine` package: `python3 -m pip install --upgrade twine`
+2. Upload to `PYPI` test env: `python3 -m twine upload --repository testpypi dist/*`
+3. Upload to `PYPI` : `python3 -m twine upload dist/*`
+
+### CI/CD by Github workflows
+
+Refer to: https://packaging.python.org/guides/publishing-package-distribution-releases-using-github-actions-ci-cd-workflows/
+
+We want a way to upload assets automatically, it can build the packages and upload them to github releases and PYPI.
+
+(For some reason we just want the workflow only publish the release to test PYPI.)
+
+```yaml
+# This is a basic workflow to help you get started with Actions
+
+name: Update the release's assets after it published
+
+# Controls when the workflow will run
+on:
+  release:
+    # The workflow will run after release published
+    types: [published]
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  # This workflow contains a single job called "build"
+  build:
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          python-version: '3.8'
+          architecture: 'x64'
+      - name: Install pypa/build
+        run: >-
+          python -m
+          pip install
+          build
+          --user
+      - name: Clean dist/
+        run: |
+          sudo rm -fr dist/*
+      - name: Build a binary wheel and a source tarball
+        run: >-
+          python -m
+          build
+          --sdist
+          --wheel
+          --outdir dist/
+          .
+      # Update target github release's assets 
+      - name: Update assets
+        uses: softprops/action-gh-release@v1
+        if: startsWith(github.ref, 'refs/tags/')
+        with:
+          files: ./dist/*
+      - name: Publish distribution 📦 to Test PyPI
+        if: contains(github.ref, 'beta') && startsWith(github.ref, 'refs/tags')
+        uses: pypa/gh-action-pypi-publish@release/v1
+        with:
+          user: __token__
+          password: ${{ secrets.TEST_PYPI_API_TOKEN }}
+          repository_url: https://test.pypi.org/legacy/
+          packages_dir: dist/
+          verify_metadata: false
+```
```

Comment:
Add this at the bottom
# Learn more about Milvus
Milvus is a powerful tool capable of powering a vast array of artificial intelligence and vector similarity search applications. To learn more about the project, check out the following resources:
- Read our [blog](https://milvus.io/blog).
- Interact with our open-source community on [Slack](https://milvusio.slack.com/join/shared_invite/zt-e0u4qu3k-bI2GDNys3ZqX1YCJ9OM~GQ#/shared-invite/email).
- Use or contribute to the world’s most popular vector database on [GitHub](https://github.com/milvus-io/milvus/).
- Quickly test and deploy AI applications with our new [bootcamp](https://github.com/milvus-io/bootcamp).